src/clusterfuzz/_internal/bot/tasks/commands.py

```diff
@@ -285,8 +285,8 @@ def process_command_impl(task_name,
     # A misconfiguration led to this point. Clean up the job if necessary.
     # TODO(ochang): Remove the first part of this check once we migrate off the
     # old untrusted worker architecture.
-    # Comment this "if" out to run a task locally.
```

Comment:
What situation should I comment this in? I was able to run fuzz task locally without touching this

Reply:
I do remember things hanging when running tasks on my own machine. I'll have to run it without this and find out.

Reply:
Yeah, that is indeed true. I take that back